crates/matrix-sdk/src/account.rs

```diff
@@ -916,6 +920,91 @@ impl Account {
         Ok(ignored_user_list)
     }
 
+    /// Set the media previews display policy for the account.
+    ///
+    /// This is an unstable event, so we always default to setting the unstable
+    /// version of the event for now.
+    pub async fn set_media_previews_setting(&self, new_setting: MediaPreviews) -> Result<()> {
+        let mut media_preview_config_to_update = if let Some(unstable_media_preview_config) =
+            self.get_unstable_media_preview_config().await?
+        {
+            unstable_media_preview_config
+        } else {
+            UnstableMediaPreviewConfigEventContent::new(Default::default(), Default::default())
+        };
```

Comment:
Would this work?

```
let mut media_preview_config_to_update = self.get_unstable_media_preview_config().await?.unwrap_or_else(|| UnstableMediaPreviewConfigEventContent::new(Default::default(), Default::default()));
```

If so, this would probably match the house style a bit more.